teal.widgets - bs345 (#84)
insightsengineering/teal.widgets: NEWS.md

# teal.widgets 0.1.1.9013

### Breaking changes

> * `panel_group` and `panel_item` are no longer offer to be optionally a shiny input.

Reviewer: I think this needs to be a little clearer - it's the css class name rather than the panel group itself?

> * `panel_group` and `panel_item` are no longer offer to be optionally a shiny input.

Author: ```suggestion
* `panel_group` and `panel_item` functions are no longer offer to be optionally a shiny input.
```

> * `panel_group` and `panel_item` functions are no longer offer to be optionally a shiny input.

Reviewer 2: ```suggestion
* Updated `panel_group` and `panel_item` functions to no longer be an optional shiny input.
```
